app/javascript/util/listingUtil.ts

```diff
@@ -538,3 +538,16 @@ export const getPriorityTypeText = (priortyType: string): string => {
   }
   return text
 }
+
+// return content to display in image tag
+export const getTagContent = (listing: RailsListing) => {
+  // Custom_Listing_Type takes precedence for deciding tag content
+  if (listing.Custom_Listing_Type) {
+    const text: string = getCustomListingType(listing.Custom_Listing_Type)
+    if (text) return [{ text }]
```

Comment:
In the future, will we support multiple custom listing types. Which would result in multiple tags on the image card?

Reply:
I havent heard a requirement for multiple tags on a single image but it could be something that is requested in the future. For now, similar to reserved community types, I am just returning a single tag.